contracts/GnosisSafePersonalEdition.sol

```diff
@@ -1,26 +1,47 @@
 pragma solidity 0.4.24;
 import "./GnosisSafe.sol";
 import "./MasterCopy.sol";
-import "./SignatureValidator.sol";
+import "./SignatureDecoder.sol";
 import "./SecuredTokenTransfer.sol";
 
+contract ISingatureValidator {
```

Comment:
This interface will be moved to a separate file when merging Personal and Team Edition